### PR TITLE
Linux fix

### PR DESCRIPTION
**Pull request by kspacewalk**
Linux fix, please test.

### CODE REVIEW

File: fsleyes/fsleyes [hunk 1]
```diff
@@ -20,17 +20,15 @@ if [ -z "$VIRTUAL_ENV" ] ; then
...
 PYSUBVER="$(python --version 2>&1 | cut -d ' ' -f2)"  # e.g., 2.7.10
 
 PYTHON_BINARY="bin/python$PYVER"
-FRAMEWORK_PYTHON="$SYSTEM_FRAMEWORK_PYTHON_ROOT/$PYTHON_BINARY"
-
-VENV_SITE_PACKAGES="$VIRTUAL_ENV/lib/python$PYVER/site-packages"
-
-# Ensure wx.pth is set up in the virtualenv
-#cp "/Library/Python/$PYVER/site-packages/wxredirect.pth" "$VENV_SITE_PACKAGES/wx.pth"
+if [[ $OSX ]]; then
```
Comment (isolovey) [inline fsleyes/fsleyes:26]: I don't think this if statement works. Please try `[[ "$OSTYPE" == "darwin"* ]];`.

**Review by isolovey — CHANGES_REQUESTED** [review]

File: fsleyes/fsleyes (hunk 1 above)
Comment (kspacewalk) [reply fsleyes/fsleyes:26]: Okay, addressed it

**Comment by kspacewalk** [comment]
Fixed OS check and also fixed which python binary gets called.

**Comment by isolovey** [comment]
Okay, looks good to me.

**Pull request merged**